litestar/app.py

```diff
@@ -210,6 +209,7 @@ def __init__(
         websocket_class: type[WebSocket] | None = None,
         lifespan: list[Callable[[Litestar], AbstractAsyncContextManager] | AbstractAsyncContextManager] | None = None,
         pdb_on_exception: bool | None = None,
+        _preferred_validation_backend: Literal["attrs", "pydantic"] | None = None,
```

Comment:
This is not ideal, but the only good way to solve this. The alternative - removing this parameter outright - makes it impossible to change during testing, since it is being referenced at the time the signature models are created, which happens inside of `__init__`.

Reply:
Still, I think it achieves the objective of not encouraging dependence on the feature, so SGTM.